src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -3977,6 +4004,16 @@ public TransformEngineChain getTransformEngineChain()
     }
 
     /**
+     * {@inheritDoc}
+     */
+    @Override
+    public REDBlock getPayloadBlock(ByteArrayBuffer baf)
```

Comment:
sure, no problem. Some changes to the JVB where necessary: https://github.com/jitsi/jitsi-videobridge/pull/504/files